pack/tcu/tsn.json

```diff
@@ -1,4 +1,39 @@
 [	
+	{
+		"code": "05109",
+		"cost": 3,
+		"deck_limit": 2,
+		"faction_code": "guardian",
+		"flavor": "I can't die here. Not now. Not yet.",
+		"illustrator": "Mauro Dal Bo",
+		"name": "Something Worth Fighting For",
+		"pack_code": "tsn",
+		"position": 109,
+		"quantity": 2,
+		"sanity": 3,
+		"skill_willpower": 1,
+		"text": "Something Worth Fighting For may be assigned horror dealt to other investigators at your location",
+		"traits": "Talent.",
+		"type_code": "asset",
+		"xp": 0
+	},
+	{
+		"code": "05110",
+		"cost": 0,
+		"deck_limit": 2,
+		"faction_code": "seeker",
+		"flavor": "...when you have eliminated all which is impossible, then whatever remains, however improbable, must be the truth. <cite>Arthur Conan Doyle, \"The Adventure of the Blanched Soldier\"",
```

Comment:
Would be nice to grab the ellipsis character for this instead of `...` -- I usually copy paste it from the wikipedia article.

Add an escaped quote (`\"`) to start of quote and after the `truth.`

I think we usually do our citation like so
`–Robert W. Chambers, \"In the Court of the Dragon,\" <u>The King in Yellow</u>",`
 but given this is a short story and not a book, I think just the quotes is sufficient?